lib/handler/status.c

```diff
@@ -112,7 +112,7 @@ static void send_response(struct st_h2o_status_collector_t *collector)
     resp[cur_resp++] = (h2o_iovec_t){H2O_STRLIT("\n}\n")};
 
     req->res.status = 200;
-    h2o_add_header(&req->pool, &req->res.headers, H2O_TOKEN_CONTENT_TYPE, H2O_STRLIT("text/plain; charset=utf-8"));
+    h2o_add_header(&req->pool, &req->res.headers, H2O_TOKEN_CONTENT_TYPE, H2O_STRLIT("application/json; charset=utf-8"));
```

Comment:
Sorry, this isn't relevant to this PR.
I believe that canonical MIME type of JSON document is `application/json`, and also this json document are consumed by only "status" web-app, so people don't see in browsers, isn't it?
https://tools.ietf.org/html/rfc7159#section-11

But, I have no strong kodawari about this, so can get it back to`text/plain` ( also with `debug_state.c` )